components/content/MultiCameraDemo.vue

```diff
@@ -0,0 +1,107 @@
+<script setup>
+import { computed } from 'vue'
+import { TresCanvas, useRenderLoop } from '@tresjs/core'
+import { useAnimations, useGLTF, useTweakPane } from '@tresjs/cientos'
+import { PerspectiveCamera, Vector4, Vector3 } from 'three'
+import { useWindowSize } from '@vueuse/core'
+
+const { width, height } = useWindowSize()
+const WIDTH = (width.value / 4) * window.devicePixelRatio
+const HEIGHT = (height.value / 4) * window.devicePixelRatio
+const ASPECT_RATIO = computed(() => width.value / height.value)
+
+const cameras = []
+
+const cameraOptions = [
+  {
+    viewPort: new Vector4(Math.floor(0), Math.floor(0), Math.ceil(WIDTH * 2), Math.ceil(HEIGHT * 2)),
+    position: new Vector3(15, 0, 3),
+    factor: 0.4,
+    name: 'left_bottom',
+  },
+  {
+    viewPort: new Vector4(Math.floor(WIDTH), Math.floor(0), Math.ceil(WIDTH * 2), Math.ceil(HEIGHT * 2)),
+    position: new Vector3(0, 0, -3),
+    factor: 2,
+    name: 'center_bottom',
+  },
+  {
+    viewPort: new Vector4(Math.floor(WIDTH * 2), Math.floor(0), Math.ceil(WIDTH * 2), Math.ceil(HEIGHT * 2)),
+    position: new Vector3(-15, 0, 3),
+    factor: 0.4,
+    name: 'right_bottom',
+  },
+  {
+    viewPort: new Vector4(Math.floor(WIDTH - 75), Math.floor(HEIGHT), Math.ceil(WIDTH * 2.5), Math.ceil(HEIGHT * 2.5)),
+    position: new Vector3(0, 0, 3),
+    factor: 2,
+    name: 'center_top',
+  },
+]
+
+cameraOptions.forEach(data => {
+  const currentCam = new PerspectiveCamera(40, ASPECT_RATIO.value, 0.1, 10)
+  currentCam.name = data.name
+  currentCam.viewport = data.viewPort
+  currentCam.position.set(...data.position)
+  currentCam.position.multiplyScalar(data.factor)
+  currentCam.lookAt(0, 0, 0)
+  currentCam.updateMatrixWorld()
+  cameras.push(currentCam)
+})
+
+const { scene: model, animations } = await useGLTF(
+  'https://raw.githubusercontent.com/Tresjs/assets/main/models/gltf/warcraft-3-alliance-footmanfanmade/source/Footman_RIG.glb',
```

Comment:
@JaimeTorrealba forgot to ask you, what are the licenses for that model, is it [C0](https://creativecommons.org/share-your-work/public-domain/cc0/)?

Reply:
Yes it's free... 

you can check it out here: https://sketchfab.com/3d-models/warcraft-3-alliance-footmanfanmade-201452e568064aedadccfafb668ef6a5